### PR TITLE
feat: resize pane, #2

### PR DESCRIPTION
**Pull request by QingWei-Li**

### CODE REVIEW

**Comment by egoist** [comment]
Can you give me the access to this repo? I need to push some patches 😅

File: src/components/Sidebar.vue [hunk 1]
```diff
@@ -22,6 +23,38 @@
       paths() {
         return this.$store.state.toys.paths
       }
+    },
+
+    created() {
+      this.boundary = {
```
Comment (egoist) [inline src/components/Sidebar.vue]: maybe define the boundary outside the component?

File: src/components/Sidebar.vue [hunk 2]
```diff
@@ -22,6 +23,38 @@
...
+        document.addEventListener('mouseup', this.handleMouseUp);
+        document.onselectstart = () => false;
+        document.ondragstart = () => false;
+      },
+
+      handleMouseMove({ clientX }) {
+        if (!this.resizing ||
+          clientX < this.boundary.min ||
+          clientX > this.boundary.max) return;
+        this.$refs.sidebar.style.width = this.finalX + clientX - this.startX + 'px';
```
Comment (egoist) [inline src/components/Sidebar.vue]: isn't the `this.finalX` equal to `this.startX` ?

**Review by egoist — CHANGES_REQUESTED** [review]

File: src/components/Sidebar.vue (hunk 2 above)
Comment (QingWei-Li) [reply src/components/Sidebar.vue]: finalX is sidebar original width, I will rename it to originWidth

**Comment by QingWei-Li** [comment]
Done

**Pull request merged**